bsoid_app/extract_features.py

```diff
@@ -16,15 +16,14 @@
 from bsoid_app.bsoid_utilities.load_workspace import load_feats, load_embeddings
 from bsoid_app.config import *
 
-
 class extract:
 
     def __init__(self, working_dir, prefix, processed_input_data, framerate):
-        st.subheader('EXTRACT AND EMBED FEATURES')
+        print('EXTRACT AND EMBED FEATURES')
         self.working_dir = working_dir
         self.prefix = prefix
         self.processed_input_data = processed_input_data
-        self.framerate = framerate
+        self.framerate = float(framerate)
```

Comment:
You do this here but not on other places. See my rec on `bsoid_app.py`

Reply:
moved it to bsoid_app